### ISSUE TITLE
can not  work for i3-wm

### ISSUE THREAD

**Issue by hellojukay** [linuxdeepin/deepin-screen-recorder#31]

**Comment by hellojukay** [comment]
![image](https://user-images.githubusercontent.com/10371891/64409213-056cf800-d0bb-11e9-85c9-6682eb7b6a70.png)

**Comment by awh6al** [comment]
the same thing here?

**Comment by weynhamz** [comment]
Yep, I love deepin-screenshot, simple get the job done, better than other alternatives.

Recently, Arch installed the replacement deepin-screen-recorder, and it is just broken in i3, has to install deepin-screenshot back and ignore it from future upgrade, surprisingly find deepin-screenshot is abandoned by Deepin team, curious about the reason, any link on this?

**Comment by RaitoBezarius** [comment]
I'm using i3 (Xorg) and deepin-screen-recorder is working for me under multi-monitor setup, what do you mean by not working?

**Comment by gkotian** [comment]
Also using i3 on Arch, but with the "upgrade" to `deepin-screen-recorder`, the UI has become super difficult to use. Here's what it looks like on my system: https://github.com/linuxdeepin/deepin-screen-recorder/issues/38#issuecomment-645007189.

I used to love the simplicity of `deepin-screenshot`, and `deepin-screen-recorder` clearly supports many more features. If the UI is fixed, it would be such an awesome tool.
